tool/tctl/common/tctl.go

```diff
@@ -327,6 +316,20 @@ func applyConfig(ccf *GlobalCLIFlags, cfg *service.Config, loadConfigExt LoadCon
 			return nil, trace.Wrap(err)
 		}
 	}
+
+	// Config file should take precedence, if available.
+	if fileConf == nil && ccf.IdentityFilePath == "" && loadConfigExt != nil {
```

Comment:
I can see a need for me personally to run `tctl` against remote clusters from my laptop, which also happens to have `/etc/teleport.yaml` present because it's used for Teleport testing. 

This might be a bit of a edge case, but I'm wondering whether we should provide some way to explicitly specify that `tctl` should use my `tsh` credentials and ignore the Teleport config file for cases like this. The alternative is always that I can just write some kind of `tctl-remote` wrapper for myself that hides the existence of `/etc/teleport.yaml`. Maybe I can just explicitly 

I'm not sure how common this would be outside of the company.

Reply:
we can have an env variable with a hint, similar to how KUBECONFIG does it

Reply:
@webvictim Can you create a ticket for this and put it in the 5.1 milestone?

Reply:
#4841